telco-hub/configuration/reference-crs/required/gitops/ex-ztp-repo-ssh.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ztp-repo
+  namespace: openshift-gitops
+  labels:
+    argocd.argoproj.io/secret-type: repository
+stringData:
```

Comment:
Can we combine these two files so that we have just one ztp-repo.yaml secret. We can include the ssh as the default and have a commented out version for http.

Reply:
done